packages/react-native-skia/components/text/NumberOfLines_ellipSizeMode.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { AppRegistry, Text, View } from 'react-native';
+
```

Comment:
should we add test app with height also defined .

Reply:
yes

Reply:
yes